components/ActionSheet/examples/ActionSheetComparisonExampleViewController.m

```diff
@@ -27,61 +28,56 @@ @interface ActionSheetComparisonExampleViewController ()
 
 @end
 
-@implementation ActionSheetComparisonExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetComparisonExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
     _colorScheme = [[MDCSemanticColorScheme alloc] init];
     _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
     _showMaterialButton = [[MDCButton alloc] init];
     _showUIKitButton = [[MDCButton alloc] init];
-    _buttonScheme = [[MDCButtonScheme alloc] init];
   }
   return self;
 }
 
-- (MDCContainerScheme *)containerScheme {
-  if (!_containerScheme) {
-    _containerScheme = [[MDCContainerScheme alloc] init];
-  }
-  _containerScheme.colorScheme = self.colorScheme;
-  _containerScheme.typographyScheme = self.typographyScheme;
-  return _containerScheme;
-}
-
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  self.view.backgroundColor = _colorScheme.backgroundColor;
-  [_showMaterialButton setTitle:@"Show Material Action sheet" forState:UIControlStateNormal];
-  [_showMaterialButton sizeToFit];
-  [_showUIKitButton setTitle:@"Show UIKit Action sheet" forState:UIControlStateNormal];
-  [_showUIKitButton sizeToFit];
-  _buttonScheme.colorScheme = _colorScheme;
-  _buttonScheme.typographyScheme = _typographyScheme;
-  [MDCContainedButtonThemer applyScheme:_buttonScheme toButton:_showMaterialButton];
-  [_showMaterialButton addTarget:self
+  if (self.containerScheme.colorScheme == nil) {
+    self.containerScheme.colorScheme = self.colorScheme;
```

Comment:
I have two concerns about this code. The first is that we're retaining `colorScheme` (and `typographyScheme`) separately from `containerScheme`. I believe the goal is to drop `colorScheme` entirely and rely solely on `containerScheme` in examples.  Second, `self.colorScheme` is null-unspecified and so could be nil as well.  

I would suggest dropping `colorScheme` entirely (we will have to do so in a follow-up PR if you don't here) and use a default MDCSemanticColorScheme if the container scheme doesn't have one.

Reply:
Done